meilisearch-http/src/analytics/segment_analytics.rs

```diff
@@ -840,6 +855,91 @@ impl DocumentsDeletionAggregator {
     }
 }
 
+#[derive(Default, Serialize)]
+pub struct TasksAggregator {
+    #[serde(skip)]
+    timestamp: Option<OffsetDateTime>,
+
+    // context
+    #[serde(rename = "user-agent")]
+    user_agents: HashSet<String>,
+
+    filtered_by_uid: bool,
+    filtered_by_index_uid: bool,
+    filtered_by_type: bool,
+    filtered_by_status: bool,
```

Comment:
Are we sure we want these names and not the plural ones?
Are we sure we don't prefer a count here, instead?

Reply:
As seen internally, we don't need that.